application/build.gradle

```diff
@@ -29,6 +29,8 @@ dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-validation'
   implementation 'org.springframework.cloud:spring-cloud-starter-aws'
   implementation 'org.springframework.cloud:spring-cloud-starter-aws-messaging'
+  implementation 'org.springframework.boot:spring-boot-starter-activemq'
+  implementation 'org.springframework:spring-jms:5.3.2'
```

Comment:
isn't the JMS dependency transitively pulled with the starter for ActiveMQ? I would try to avoid hard-coding Spring dependency versions to make sure Spring Boot manages them for us

Reply:
Yes, you're right. I've removed the JMS dependency here.